packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -288,6 +288,9 @@ class Auth {
         storage.delete(key)
     }
 
+    /**
+     * Gets the status of the Do-Not-Track (DNT) local state, representing user choice.
```

Comment:
Can we also comment on what its possible outputs are and what they mean? Especially when it returns undefined.